infra/k8s/kops-weave/weave.yml

```diff
@@ -177,7 +191,8 @@ items:
                     fieldRef:
                       apiVersion: v1
                       fieldPath: spec.nodeName
-              image: 'docker.io/weaveworks/weave-kube:2.6.0'
+              #image: 'docker.io/weaveworks/weave-kube:2.6.0'
+              image: 'nonsens3/weave-kube'
```

Comment:
@raulk @Stebalien note that this is using a forked version of the weave CNI plugin based on https://github.com/weaveworks/weave/pull/3754